### PR TITLE
[ONE]Implementing Sword of Forge and Frontier

### PR DESCRIPTION
**Pull request by AhmadYProjects**

### CODE REVIEW

**Comment by AhmadYProjects** [comment]
[[Sword of Forge and Frontier]]

File: Mage.Sets/src/mage/cards/s/SwordOfForgeAndFrontier.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        this.subtype.add(SubType.EQUIPMENT);
+
+        // Equipped creature gets +2/+2 and has protection from red and from green.
+        Ability ability = new SimpleStaticAbility(new BoostEquippedEffect(2,2));
+        ability.addEffect(new GainAbilityAttachedEffect(
+                ProtectionAbility.from(ObjectColor.GREEN,ObjectColor.RED), AttachmentType.EQUIPMENT).setText(
+                "and has protection from red and from green"));
+        this.addAbility(ability);
+        // Whenever equipped creature deals combat damage to a player, exile the top two cards of your library. You may play those cards this turn. You may play an additional land this turn.
+        Ability ability2 = new DealsDamageToAPlayerAttachedTriggeredAbility(new PlayAdditionalLandsControllerEffect(1, Duration.EndOfTurn),"equipped creature", false);
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/s/SwordOfForgeAndFrontier.java:38]: you should switch the two effects here since they're in the opposite order

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/s/SwordOfForgeAndFrontier.java (hunk 1 above)
Comment (AhmadYProjects) [reply Mage.Sets/src/mage/cards/s/SwordOfForgeAndFrontier.java]: I changed it but why does the order matter?

Comment (alexander-novo) [reply Mage.Sets/src/mage/cards/s/SwordOfForgeAndFrontier.java]: The order of the effects will affect the order in the text on the card. Ideally it should match the actual text on the card.

**Pull request merged**